app/interactors/deployers/engineyard.rb

```diff
@@ -23,10 +23,13 @@ def last_deploy_commit
 
     # https://github.com/engineyard/engineyard/blob/v3.0.1/lib/engineyard/cli.rb#L123
     def deploy(deploy, options={})
+      config = { maintenance_on_migrate: options[:maintenance_page] }
+      config[:precompile_assets] = options[:precompile_assets] if options.key? :precompile_assets
+      config[:precompile_unchanged_assets] = options[:precompile_unchanged_assets] if options.key? :precompile_unchanged_assets
       options = {
         ref: deploy.branch,
         environment: environment_name,
-        config: { maintenance_on_migrate: options[:maintenance_page] }
+        config: config }
```

Comment:
Ah... here it is. I had to pop both of those off b/c there was a syntax error in one of them that crashed Houston. Looks like it's the extra `}` here.